src/storybook/story-test-runner/extend-stories.ts

```diff
@@ -102,21 +120,36 @@ function getStoryFile(storyPath: string, { requireFn = require } = {}): StoryFil
             loggedStoryFileRequireError = true;
 
             const warningMessage = [
-                `"testplane" section is ignored in storyfile "${storyPath}",`,
-                `because the file could not be read:\n${error}`,
-                "\nThere could be other story files.",
-                "\nSet 'TESTPLANE_STORYBOOK_DISABLE_STORY_REQUIRE_WARNING' environment variable to hide this warning",
+                `Testplane custom section is ignored in storyfile "${storyPath}",`,
+                `because the file could not be read:\n${error}\n${disableWarningsMessage}`,
             ].join(" ");
 
             console.warn(warningMessage);
         }
     }
 
-    const isStoryFileFailedToLoad = !storyFile || typeof storyFile === "function";
-
     unmockFn();
 
-    return isStoryFileFailedToLoad ? null : storyFile;
+    const isStoryFileFailedToLoad = !storyFile || isMockedModuleProxy(storyFile);
+
+    if (isStoryFileFailedToLoad) {
+        return null;
+    }
+
+    const ignoredProperties = getNestedMockedPropertiesPaths(storyFile);
```

Comment:
We can't remove nested mocked props here as module's properties are not configurable